src/app/App.tsx

```diff
@@ -38,11 +38,14 @@ const useInitApp = () => {
   return { isLoading: !initialized };
 };
 
+/*
+    <Router basename={process.env.PUBLIC_URL}>
+*/
 export default function App() {
   const { isLoading } = useInitApp();
 
   return (
-    <Router basename={process.env.PUBLIC_URL}>
+    <Router>
```

Comment:
This kind of change is actually opposed to a minor goal of Europa, that is, to not make the assumption that Europa has control over the absolute routes. Therefore, in the future, if for some reason (perhaps when it is replaced with something new), it can be served from the same host as the new thing, making cookie management simpler.